src/test/java/com/microsoft/sqlserver/jdbc/databasemetadata/DatabaseMetadataGetIndexInfoTest.java

```diff
@@ -0,0 +1,174 @@
+    private static String col1Name = AbstractSQLGenerator.escapeIdentifier("p1");
+    private static String col2Name = AbstractSQLGenerator.escapeIdentifier("p2");
+    private static String col3Name = AbstractSQLGenerator.escapeIdentifier( "p3");
+    
+    @BeforeAll
+    public static void setupTests() throws Exception {
+        setConnection();
+    }
+    
+    @BeforeEach
+    public void init() throws Exception {
+        try (Connection con = getConnection()) {
+            con.setAutoCommit(false);
+            try (Statement stmt = con.createStatement()) {
+            	TestUtils.dropTableIfExists(tableName, stmt);
+            	String createTableSQL = "CREATE TABLE " + tableName + " (" +
+                        col1Name + " INT, " +
+                        col2Name + " INT, " +
+                        col3Name + " INT)";
+            	
+            	stmt.executeUpdate(createTableSQL);
+            	assertNull(connection.getWarnings(), "Expecting NO SQLWarnings from 'create table', at Connection.");
+            	assertNull(stmt.getWarnings(), "Expecting NO SQLWarnings from 'create table', at Statement.");
+                
+                String createClusteredIndexSQL = "CREATE CLUSTERED INDEX IDX_Clustered ON " + tableName + "(" + col1Name + ")";
+    			stmt.executeUpdate(createClusteredIndexSQL);
+    			assertNull(connection.getWarnings(), "Expecting NO SQLWarnings from 'create index', at Connection.");
+                assertNull(stmt.getWarnings(), "Expecting NO SQLWarnings from 'create index', at Statement.");
+    			
+    			String createNonClusteredIndexSQL = "CREATE NONCLUSTERED INDEX IDX_NonClustered ON " + tableName + "(" + col2Name + ")";
+    			stmt.executeUpdate(createNonClusteredIndexSQL);
+    			assertNull(connection.getWarnings(), "Expecting NO SQLWarnings from 'create index', at Connection.");
+                assertNull(stmt.getWarnings(), "Expecting NO SQLWarnings from 'create index', at Statement.");
+                
+                String createColumnstoreIndexSQL = "CREATE COLUMNSTORE INDEX IDX_Columnstore ON " + tableName + "(" + col3Name + ")";
+    			stmt.executeUpdate(createColumnstoreIndexSQL);
+            	assertNull(connection.getWarnings(), "Expecting NO SQLWarnings from 'create index', at Connection.");
+                assertNull(stmt.getWarnings(), "Expecting NO SQLWarnings from 'create index', at Statement.");
+            }
+            con.commit();
+        }
+    }
+    
+    @AfterEach
+    public void terminate() throws Exception {
+        try (Connection con = getConnection(); Statement stmt = con.createStatement()) {
+            try {
+                TestUtils.dropTableIfExists(tableName, stmt);
+            } catch (SQLException e) {
+                fail(TestResource.getResource("R_unexpectedException") + e.getMessage());
+            }
+        }
+    }
+
+    @Test
+    public void testGetIndexInfo() throws SQLException, SQLServerException {

```

Comment:
Exception not needed.
```suggestion
    public void testGetIndexInfo() throws SQLException {
```